recipes/perl-uri/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "perl-uri" %}
+{% set version = "5.17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/O/OA/OALDERS/URI-5.17.tar.gz
+  sha256: 5f7e42b769cb27499113cfae4b786c37d49e7c7d32dbb469602cd808308568f8
+
+build:
```

Comment:
To ensure that the module version matches your Perl version at runtime, add a weak run export  to the build section like so:

```
build:
  [...]
  run_exports:
    weak:
      - {{ name }} ={{ version }}
```